### PR TITLE
Update incorrect geolocation description

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: src/site/content/en/blog/native-hardware-user-location/index.md [hunk 1]
```diff
@@ -408,11 +408,10 @@ Once you've got geolocation set up, you'll want to:
...
 You can do both from Chrome DevTools.
 
-[Open Chrome DevTools](/web/tools/chrome-devtools/#open) and then
-[open the Console Drawer](/web/tools/chrome-devtools/console/#open_as_drawer).
-
-[Open the Console Drawer menu](/web/tools/chrome-devtools/settings#drawer-tabs)
-and click the **Sensors** option to show the Sensors Drawer.
+1. [Open Chrome DevTools](/web/tools/chrome-devtools/#open).
+2. Press `Esc` to [Open the Console Drawer](/web/tools/chrome-devtools/console/#open_as_drawer).
+3. [Open the Console Drawer menu](/web/tools/chrome-devtools/settings#drawer-tabs)
```
Comment (tomayac) [inline src/site/content/en/blog/native-hardware-user-location/index.md:413]:
```suggestion
2. Press <kbd>Esc</kbd> to [Open the Console drawer](/web/tools/chrome-devtools/console/#open_as_drawer).
3. [Open the Console drawer menu](/web/tools/chrome-devtools/settings#drawer-tabs)
```

File: src/site/content/en/blog/native-hardware-user-location/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: User Location
-subhead: A new font specification that can significantly reduce font file sizes
+subhead: The Geolocation API lets you discover, with the user's consent, the user's location.
```
Comment (tomayac) [inline src/site/content/en/blog/native-hardware-user-location/index.md:4]:
```suggestion
subhead: The Geolocation API lets you discover the user's location, given their consent.
```

**Review by tomayac — APPROVED** [review]
LGTM, with a few suggestions.

**Comment by tomayac** [comment]
Careful, https://github.com/GoogleChrome/web.dev/pull/9116 renames the file to just `/user-location/`! You need to reflect this change.

**Comment by tomayac** [comment]
> Careful, #9116 renames the file to just `/user-location/`! You need to reflect this change.

Ah, ignore, it's part of that other PR.

**Comment by tomayac** [comment]
@jecfish Erm, you ignored all the suggestions?

**Comment by jecfish** [comment]
Oh no, sorry, I read the last message from email and click "Approve". I added the fixes in #9116 9116

**Pull request merged**